source/command-monitoring/tests/bulkWrite.yml

```diff
@@ -53,7 +53,7 @@ tests:
       arguments:
         requests:
           - insertOne: 
-              document: { _id: 4, x: 44 }
+              document: { _id: -4, x: 44 }
```

Comment:
Can you make the _id the name of this test `"bulkWriteInsertW0"`? That reduces the chance we'll accidentally add the same _id in a future test.

I'd prefer a real solution that waits for the w0 write to complete but I'm not sure of a good way to make that generic.

Reply:
> Changing the collection_name is another option, also simple.

No harm in doing both.

Reply:
Agreed.  To make it clearer I split bulkWrite.yml into two files, one for the w:1 test and one for the w:0 test.